google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java

```diff
@@ -61,13 +64,31 @@ public void actionPerformed(AnActionEvent e) {
   /** The wizard for the "Add Cloud Libraries" menu action. */
   private static final class AddCloudLibrariesWizard extends AbstractWizard<Step> {
 
+    private final SelectClientLibrariesStep selectClientLibrariesStep;
+    private final ManageCloudApisStep manageCloudApisStep;
+
     AddCloudLibrariesWizard(Project project) {
       super(GctBundle.message("cloud.libraries.dialog.title"), project);
-      addStep(new SelectClientLibrariesStep());
-      addStep(new ManageCloudApisStep());
+
+      selectClientLibrariesStep = new SelectClientLibrariesStep(project);
+      selectClientLibrariesStep.addModuleSelectionListener(event -> updateButtons());
```

Comment:
why not add this listener directly in the `GoogleCloudApiSelectorPanel`?

Reply:
What do you mean? The call to `updateButtons()` is specific to this `AbstractWizard<Step>`, which the `GoogleCloudApiSelectorPanel` has no knowledge of. IMO exposing a way to add a custom listener for when a module is selected (like this) prevents the panel from knowing too much about the context in which it's created.

Reply:
oops. I didn't realize that. sgtm